faculty_cli/cli.py

```diff
@@ -1451,3 +1450,147 @@ def dataset_ls(project, prefix, show_hidden):
         prefix, project_id=project_id, show_hidden=show_hidden
     ):
         click.echo(item)
+
+
+@cli.group()
+def template():
+    """Manipulate templates in the Faculty knowledge centre."""
+    pass
+
+
+@template.command()
+def init():
+    """Create a blank template."""
+    print("hello")
+
+
+@template.command()
+@click.argument(
+    "source_template_name"
+)  # Outstanding question as to whether this should also alllow the template_ID
+@click.option(
+    "--version",
+    default=None,
+    help="The specific version of the template to use.",
+)
+@click.option(
```

Comment:
Our CLI design has this argument as a positional, rather than a flag-based argument (i.e. a `click.argument` with a default set rather than a `click.option`). It should probably go above the `--version` option.